modules/preview-react/_examples/stories/examples/Table/WithSelectableRows.tsx

```diff
@@ -125,15 +125,15 @@ export const SelectableRows = () => {
       </Heading>
       <Table aria-labelledby={headingID}>
         <Table.Row gridTemplateColumns="3.5rem repeat(2, 1fr)">
```

Comment:
I think this `<Table.Row/>` should be wrapped with a `<Table.Head/>` since it's the header.

```suggestion
       <Table.Head>
        <Table.Row gridTemplateColumns="3.5rem repeat(2, 1fr)">
```